t/plugin/log-rotate.t

```diff
@@ -105,6 +106,7 @@ start xxxxxx
 --- config
     location /t {
         content_by_lua_block {
+            local io = require("io")
```

Comment:
I think you will have to revert these changes.

Reply:
It fails lint check when I try to use global "io"

Reply:
Similar to this one https://github.com/apache/apisix/actions/runs/5218439289/jobs/9419350930?pr=9619#step:4:277

Reply:
You have imported "io" but haven't used it.

Reply:
I have used it below in the for loop

Reply:
Strange, I don't see any other additions except importing "io" module in to this file 🤔